dspace-api/pom.xml

```diff
@@ -889,6 +889,14 @@
             <version>20180130</version>
         </dependency>
 
+        <!-- Useful for testing command-line tools -->
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.19.0</version>
+            <scope>test</scope>
```

Comment:
While I'm not against adding this, it looks (at a glance) like this library is now _possibly_ unmaintained. It hasn't been updated in a few years...

I stumbled on this updated alternative: https://github.com/webcompere/system-stubs (which appears to fork from this library), described more at https://www.baeldung.com/java-system-stubs

All that said, I'm not suggesting we *have* to change this dependency in this PR... I just wanted to call it out, as we may want to create a ticket to replace this later on at the very least.  So, I'd be interested in your thoughts on this.

Reply:
It seems that System Rules has effectively split.  System Rules works with JUnit 4, while [System Lambda](https://github.com/stefanbirkner/system-lambda) apparently requires something else, probably JUnit 5, and has had recent updates.  Meanwhile, System Stubs appears to work with either JUnit 4 or 5 and depends on Mockito Inline.  System Lambda doesn't work with Java 17, while System Stubs does.

So, System Stubs looks like a good fit, and we may *need* to move to it if we are to support Java 17+.  I will create such an Issue.